save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/Contest.kt

```diff
@@ -0,0 +1,56 @@
+package com.saveourtool.save.entities
+
+import com.saveourtool.save.utils.EnumType
+import com.saveourtool.save.utils.LocalDateTime
+
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+
+/**
+ * @property name organization
+ * @property status
+ * @property startTime the time contest starts
+ * @property endTime the time contest ends
+ * @property description
+ */
+@Entity
+@Serializable
```

Comment:
Please don't mix `Entity` and `Serializable`, for Project it's just legacy and for `Organization` it was required in order not to refactor `Project` class.

Reply:
Ok, will move `Contest` entity to `jvmMain` section then and will remove `Serializable`.